src/clusterd/src/bin/clusterd.rs

```diff
@@ -101,58 +117,42 @@ struct Args {
 #[tokio::main]
 async fn main() {
     let args = cli::parse_args(CliConfig {
-        env_prefix: Some("STORAGED_"),
+        env_prefix: Some("CLUSTERD_"),
         enable_version_flag: true,
     });
     if let Err(err) = run(args).await {
-        eprintln!("storaged: fatal: {:#}", err);
+        eprintln!("clusterd: fatal: {:#}", err);
         process::exit(1);
     }
 }
 
-fn create_communication_config(args: &Args) -> Result<timely::CommunicationConfig, anyhow::Error> {
-    let threads = args.workers;
-    if threads > 1 {
-        Ok(timely::CommunicationConfig::Process(threads))
-    } else {
-        Ok(timely::CommunicationConfig::Thread)
-    }
-}
-
-fn create_timely_config(args: &Args) -> Result<timely::Config, anyhow::Error> {
-    Ok(timely::Config {
-        worker: timely::WorkerConfig::default(),
-        communication: create_communication_config(args)?,
-    })
-}
-
 async fn run(args: Args) -> Result<(), anyhow::Error> {
     mz_ore::panic::set_abort_on_panic();
     let (tracing_target_callbacks, _sentry_guard) = mz_ore::tracing::configure(
-        "storaged",
+        "clusterd",
         &args.tracing,
         mz_service::tracing::mz_sentry_event_filter,
         (BUILD_INFO.version, BUILD_INFO.sha, BUILD_INFO.time),
     )
     .await?;
-
-    // Keep this _after_ the mz_ore::tracing::configure call so that its panic
-    // hook runs _before_ the one that sends things to sentry.
```

Comment:
What's up with this comment? Does it not apply anymore?

Reply:
It does! Just got lost in the refactor. Good eyes, thanks! Restored.